packages/showcases/css/stories/components/structure/list/examples/small.html

```diff
@@ -136,3 +136,40 @@
     </li>
   </ul>
 </div>
+
+<div class="block">
+  <ul class="vtmn-list">
+    <li
+      class="vtmn-list_item-size--small vtmn-list_item--without-divider"
+      tabindex="0"
+    >
+      <a href="/">
+        <div class="vtmn-list_text">
+          <span>List Item without divider</span>
```

Comment:
Are `span` mandatory for the style here? I have the impression that they don't bring anything else, no?

Reply:
yes it's mandatory, the span are need if the listItem are secondary text under
![image](https://user-images.githubusercontent.com/2856778/211038557-922fa6fd-9ab0-4604-9d7d-69e7e9e361e9.png)

Reply:
ok :/

Reply:
ok :/